pkg/systemd/host.js

```diff
@@ -148,7 +148,9 @@ function ServerTime() {
         client.call(timedate.path,
                     "org.freedesktop.DBus.Properties", "Get", [ "org.freedesktop.timedate1", "NTPSynchronized" ]).
             fail(function (error) {
-                if (error.name != "org.freedesktop.DBus.Error.UnknownProperty")
+                console.log(error);
```

Comment:
Below the messages are quietened down, and yet this logs everything. Is that intentional?